Runtime/HairInstanceBuilder.cs

```diff
@@ -150,7 +150,7 @@ public static unsafe void BuildMeshRoots(Mesh meshRoots, int strandCount, Vector
 
 				// apply to mesh
 				var meshVertexCount = strandCount;
-				var meshUpdateFlags = MeshUpdateFlags.DontNotifyMeshUsers | MeshUpdateFlags.DontRecalculateBounds | MeshUpdateFlags.DontResetBoneBounds | MeshUpdateFlags.DontValidateIndices;
```

Comment:
Note: Don't actually merge this, contact #devs-playstation about why these update flags produce invalid mesh data.